ucb-rit/coldfront
# Issue 610: Superusers can create standalone clusters from UI

Majority code from @haudquaquam 

**Changes:**
- Adds request wizard for creating standalone clusters. The form can be found at `/project/new-standalone-cluster-request/` and a link to it can be found on the admin menu.
- Views in new_project_views/request_views.py
- Forms in new_project_forms/request_forms.py
- Templates under project/project_request/standalone_cluster folder
- The wizard asks for the name and description of the cluster -> an existing PI -> if no existing PI provided, a new PI's info -> an existing manager -> if no existing manager provided, a new manager's info -> a review-and-submit page.
- Back-end:
   - Creates a Resource object and capitalizes the given name for that object.
   - Creates a Project object and ProjectUsers using the logic from the `projects create` subcommand
     - Some of the `projects create` subcommand's logic has been refactored into `projects.utils_.new_project_utils`

**Testing**
- Manual testing wizard:
  - Navigate to `/project/new-standalone-cluster-request/`, a link to it is found on the admin menu as  "Create Standalone Cluster."
  - Go through the cluster creation
  - To test new PI and/or manager creation, do not select a PI/manager from the dropdowns. The next page will then prompt you to enter user info.
  - After successful submission, go to admin dashboard. Check to see that new Project and Resource objects were created.
